content/en/community/hackathons/sessions/contributing-to-unikraft/content/Overview_Warm-Up.md

```diff
@@ -1,11 +1,15 @@
 Let's check the integrity of this library using its test suite through the exposed wrapper function.
 
-For this task, you have to move, [or clone](https://github.com/unikraft/lib-libhogweed), the library in the `$UK_LIBS` folder and the `work/01-tut-porting/apps/app-libhogweed` application in the `$UK_APPS` folder.
-Fill the `TODO` lines from the application code: add the `libhogweed` library as a dependency in its `Makefile` and call from `main.c` the function exposed by the library for running the test suite.
+For this task, you have to move the library in the `$UK_LIBS` folder and the `work/01-tut-porting/apps/app-libhogweed` application in the `$UK_APPS` folder.
+You can also keep the current directory structure but set `$UK_LIBS` and `$UK_APPS` to `work/01-tut-porting/libs` and `work/01-tut-porting/apps`.
+Fill the `TODO` lines from the application code: add the `libhogweed` library as a dependency in its `Makefile` and call the function exposed by the library for running the test suite from `main.c`.
+Remember to include the library header fine.
```

Comment:
```suggestion
Remember to correctly include the library header.
```

Reply:
This was ment to be
```
Remember to include the library header file.
```
not fine, thanks :)